lib/rides.js

```diff
@@ -1,119 +1,149 @@
 export const rides = [
```

Comment:
Consider whether 'id' is really necessary. Perhaps the index or slug can be used for identification.